suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -372,16 +372,16 @@ export const DemoScreen = () => {
                     <Box marginTop="medium">
                         <Text variant="titleMedium">Token Icons</Text>
                         <Box flexWrap="wrap" flexDirection="row">
-                            {Object.keys(tokenIcons).map((icon: string) => (
+                            {Object.keys(tokenIcons).map((iconContract: string) => (
                                 <Box
-                                    key={icon}
+                                    key={iconContract}
                                     marginRight="large"
                                     marginBottom="large"
                                     justifyContent="center"
                                     alignItems="center"
                                 >
-                                    <CryptoIcon symbol={icon as TokenSymbol} />
-                                    <Text>{icon}</Text>
+                                    <CryptoIcon symbol={iconContract as TokenAddress} />
+                                    <Text>{iconContract}</Text>
```

Comment:
Doesn't make sense anymore as contract is super unreadable and we should ideally want something like symbol + contract string, but I'm happy to leave it like this. Also expect the UI to be pretty shit with those long contract strings